packages/infection-monkey/PKGBUILD

```diff
@@ -0,0 +1,64 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=infection-monkey
+_pkgname=monkey
+pkgver=v1.8.2.r2596.g2bc0f739
+pkgrel=1
+pkgdesc='Automated security testing tool for networks.'
+arch=('any')
+groups=('blackarch' 'blackarch-networking')
+url='https://github.com/guardicore/monkey'
+depends=('python' 'python-cryptography' 'python-cffi'
+         'python-ecdsa' 'pyinstaller' 'impacket'
+	       'python-ipaddress' 'python-netifaces' 'python-odict'
+	       'python-paramiko' 'python-psutil' 'python-pycryptodome'
+         'python-pyftpdlib' 'pymssql' 'python-pypykatz'
+         'python-pysmb' 'python-requests' 'python-urllib3' 
+         'python-simplejson' 'python-flask-pymongo' 'python-flask-restful'
+         'python-boto3' 'python-botocore' 'python-pyjwt'
+         'python-flask' 'python-gevent' 'python-jsonschema'
+         'python-mongoengine' 'python-mongomock' 'python-pytest'
+         'python-dateutil' 'python-requests-mock' 'python-ring'
+         'python-stix2' 'python-six' 'python-tqdm' 
+         'python-virtualenv' 'python-werkzeug' 'python-wheel'
+         'python-flask-jwt-extended' 'python-awscli' 'python-dpath'
+         'python-stix2-patterns')
+makedepends=('git')
+license=('GPL3')
+source=("git+https://github.com/guardicore/$_pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $_pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+package() {
+  cd $_pkgname
+
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+  install -dm 755 "$pkgdir/usr/bin"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
```

Comment:
copy the `docs` folder too

Reply:
i should copy it with the `cp` command instead of `install`, right ?

Reply:
no you could do like that https://github.com/BlackArch/blackarch-pkgbuilds/blob/aed7e0a8b13b0bc4f80b49f3c9bad60816f8c5cb/PKGBUILD-generic-jointscript#L37